### PR TITLE
Version 0.30.2

### PR DESCRIPTION
**Pull request by vvanglro**
# Summary

# Checklist

- [ ] I understand that this PR may be closed in case there was no previous discussion. (This doesn't apply to typos!)
- [ ] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
- [ ] I've updated the documentation accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+## 0.30.2 (2024-07-03)
+
+### Added
+
+- Add reason support on WebSocketDisconnectEvent (#2324)
+
+### Fixed
+
+- Iterate subprocesses in-place (#2373)
```
Comment (Kludex) [inline CHANGELOG.md]:
```suggestion
- Iterate subprocesses in-place on the process manager (#2373)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## 0.30.2 (2024-07-03)
```
Comment (Kludex) [inline CHANGELOG.md]:
```suggestion
## 0.30.2 (2024-07-05)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## 0.30.2 (2024-07-05)
+
+### Added
+
+- Add reason support on WebSocketDisconnectEvent (#2324)
```
Comment (Kludex) [inline CHANGELOG.md]:
```suggestion
- Add reason support to WebSocketDisconnectEvent (#2324)
```

**Comment by hadican** [comment]
When will this be released? 🤔

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## 0.30.2 (2024-07-05)
```
Comment (Kludex) [inline CHANGELOG.md]:
```suggestion
## 0.30.2 (2024-07-17)
```

**Review by Kludex — APPROVED** [review]

**Comment by hadican** [comment]
@Kludex one of the test suites has failed.

**Comment by vvanglro** [comment]
All passed.

**Pull request merged**